docs/sf/providers/aws/guide/mcp.md

```
@@ -254,124 +261,227 @@ Progress also makes the client's experience better long before the idle bound ma
-      environment:
-        SERVERLESS_MCP_PUBLIC_BASE_URL: https://mcp.example.com
-      auth:
+      authorizer: verifyToken
+      oauthDiscovery:
         issuer: https://example.us.auth0.com
-        audiences:
-          - https://mcp.example.com
 ```
 
-### Rejecting before invoke
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)